src/main/groovy/org/octopusden/octopus/license/management/plugins/gradle/LicenseGradlePlugin.groovy

```diff
@@ -37,12 +38,13 @@ class LicenseGradlePlugin implements Plugin<Project> {
         }
     }
 
-    private boolean isTrue(Project project, String property) {
-        return 'true'.equalsIgnoreCase(project.rootProject.findProperty(property) as String ?: 'true')
+    private boolean isFalse(Project project, String property) {
+        return 'false'.equalsIgnoreCase(project.rootProject.findProperty(property) as String ?: 'true') ||
+                'false'.equalsIgnoreCase(MavenParametersUtils.getProjectProperty(project, property) ?: 'true')
     }
 
     private boolean nodeOnlyIf(Project project) {
-        return !isTrue(project, 'license.skip') && !isTrue(project, NODE_SKIP_PROPERTY) && !project.gradle.startParameter.offline
+        return isFalse(project, 'license.skip') && isFalse(project, NODE_SKIP_PROPERTY) && !project.gradle.startParameter.offline
```

Comment:
semantically !isTrue is not equal isFalse. This is as 'not allowed' != 'forbidden'